src/Components/AffectedClustersTable/AffectedClustersTable.spec.ct.js

```diff
@@ -27,6 +32,51 @@ Cypress.Commands.add('getToggleCheckboxText', () =>
   cy.get(AFFECTED_LIST_TABLE).find(TOOLBAR).find('#toggle-checkbox-text')
 );
 
+function filterData(text = '') {
+  // FIXME: is this the right way to use loadash?
```

Comment:
I didn't know that Cypress exposes lodash :-D I think yes, this is another legit way to use lodash. 👍🏼 

Just in case, you also can import lodash functions directly here in any of the JS files (modules) since `ocp-advisor` has a lodash dependency and imported from `node_modules`.

e.g.
```
import camelCase from 'lodash/camelCase'
...
const abc = camelCase(...);

Reply:
Yes, it is a legit way of importing lodash. However, this is not a good import practice, since the whole library of lodash is loaded into the build. You have to to use the following notation instead:
```
import camelCase from 'lodash/camelCase';
...
const foobar = camelCase(...);
```

Reply:
It seems that I need to use
```
import camelCase from 'lodash';
...
const foobar = camelCase(...);
```
otherwise it complains. Will this be good enough or am I doing something wrong?

Reply:
hm, in fact, it must not complain. however, given that this is rather a nit-pick, feel free to use this last option that you used. i will take a look later on if I can fix it. @ikerreyes